### PR TITLE
Mofidy API group && upgrade API version

### PR DESCRIPTION
**Pull request by yaocw2020**
issue: https://github.com/rancher/harvester/issues/651
- Modify API group from `harvester.cattle.io` to `harvesterhci.io`.
- Upgrade API version from `v1alpha1` to `v1beta1`.
- Delete unused field `stopOnce` of `monitor.Monitor` structure.

### CODE REVIEW

File: pkg/apis/network.harvester.cattle.io/v1alpha1/zz_generated_list_types.go [hunk 1]
```diff
@@ -1,59 +0,0 @@
-/*
-Copyright 2019 Harvester Network Controller Authors
```
Comment (gitlawr) [inline pkg/apis/network.harvester.cattle.io/v1alpha1/zz_generated_list_types.go]: nit: move these generated file changes to the generated-code commit

**Review by gitlawr — APPROVED** [review]
LGTM

File: pkg/codegen/main.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 	controllergen "github.com/rancher/wrangler/pkg/controller-gen"
 	"github.com/rancher/wrangler/pkg/controller-gen/args"
 
-	networkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/apis/network.harvester.cattle.io/v1alpha1"
+	networkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/apis/network.harvesterhci.io/v1beta1"
```
Comment (guangbochen) [inline pkg/codegen/main.go:9]: \s\networkv1alpha1\harvnetworkv1

File: pkg/config/context.go [hunk 3]
```diff
@@ -21,8 +21,8 @@ import (
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/record"
 
-	networkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/apis/network.harvester.cattle.io/v1alpha1"
-	ctlnetwork "github.com/rancher/harvester-network-controller/pkg/generated/controllers/network.harvester.cattle.io"
+	networkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/apis/network.harvesterhci.io/v1beta1"
```
Comment (guangbochen) [inline pkg/config/context.go:24]: ditto

File: pkg/controller/agent/nad/controller.go [hunk 4]
```diff
@@ -11,7 +11,7 @@ import (
 	"k8s.io/klog"
 
 	"github.com/rancher/harvester-network-controller/pkg/config"
-	"github.com/rancher/harvester-network-controller/pkg/generated/controllers/network.harvester.cattle.io/v1alpha1"
+	ctl "github.com/rancher/harvester-network-controller/pkg/generated/controllers/network.harvesterhci.io/v1beta1"
```
Comment (guangbochen) [inline pkg/controller/agent/nad/controller.go:14]: \s\ctl\ctlharvnetworkv1

File: pkg/controller/agent/nodenetwork/controller.go [hunk 5]
```diff
@@ -16,10 +16,10 @@ import (
 	"k8s.io/client-go/tools/record"
 	"k8s.io/klog"
 
-	networkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/apis/network.harvester.cattle.io/v1alpha1"
+	networkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/apis/network.harvesterhci.io/v1beta1"
 	"github.com/rancher/harvester-network-controller/pkg/config"
 	"github.com/rancher/harvester-network-controller/pkg/controller/common"
-	"github.com/rancher/harvester-network-controller/pkg/generated/controllers/network.harvester.cattle.io/v1alpha1"
+	ctl "github.com/rancher/harvester-network-controller/pkg/generated/controllers/network.harvesterhci.io/v1beta1"
```
Comment (guangbochen) [inline pkg/controller/agent/nodenetwork/controller.go:22]: ditto

File: pkg/controller/common/nodenetwork.go [hunk 6]
```diff
@@ -9,14 +9,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 
-	networkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/apis/network.harvester.cattle.io/v1alpha1"
-	ctlnetworkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/generated/controllers/network.harvester.cattle.io/v1alpha1"
+	networkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/apis/network.harvesterhci.io/v1beta1"
+	ctl "github.com/rancher/harvester-network-controller/pkg/generated/controllers/network.harvesterhci.io/v1beta1"
```
Comment (guangbochen) [inline pkg/controller/common/nodenetwork.go:13]: ditto

File: pkg/controller/manager/clusternetwork/controller.go [hunk 7]
```diff
@@ -9,11 +9,11 @@ import (
...
 	"k8s.io/klog"
 
-	"github.com/rancher/harvester-network-controller/pkg/apis/network.harvester.cattle.io/v1alpha1"
-	networkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/apis/network.harvester.cattle.io/v1alpha1"
+	"github.com/rancher/harvester-network-controller/pkg/apis/network.harvesterhci.io/v1beta1"
+	networkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/apis/network.harvesterhci.io/v1beta1"
 	"github.com/rancher/harvester-network-controller/pkg/config"
 	"github.com/rancher/harvester-network-controller/pkg/controller/common"
-	ctlv1alpha1 "github.com/rancher/harvester-network-controller/pkg/generated/controllers/network.harvester.cattle.io/v1alpha1"
+	ctl "github.com/rancher/harvester-network-controller/pkg/generated/controllers/network.harvesterhci.io/v1beta1"
```
Comment (guangbochen) [inline pkg/controller/manager/clusternetwork/controller.go:16]: duplicate `network.harvesterhci.io/v1beta1` and ditto

**Comment by guangbochen** [comment]
let's use the similar library naming standard of the Harvester repo for better maintenance purpose, thanks

**Review by guangbochen — APPROVED** [review]

**Pull request merged**